signalfx/signalflow/computation.py

```diff
@@ -205,21 +205,20 @@ def _process_info_message(self, message):
         """Process an information message received from the computation."""
         # Extract the output resolution from the appropriate message, if
         # it's present.
+        contents = message.get('contents', None)
```

Comment:
I'm unclear on this part. The docs say `contents` is optional and imply it's present for certain kinds of messages. Below, the kinds of messages where we access values of `contents`, it is (hopefully?) guaranteed to be present. In cases where it would crash, that access would be invalid anyway (there wouldn't be any data to retrieve, why are we checking it?)

Reply:
`message` isn't `None`, so we should get either `message['contents']` here or `None` with no crash?

If `contents` is `None`, it's expected that in the cases where it's accessed below, `contents` is a provided value. The docs say it's optional but imply that it's present for the types of message that use it. They don't make it clear when it's guaranteed to be there and when it isn't, so we could be super defensive and use `contents.get('resolutionMs')` and such below if we wanted to, but this seemed to correspond with the expected data

Reply:
Ah, I believe I misunderstood this on my first pass because the change is subtle. This fixes `matchedSize` and `limitSize` which were not looking in `contents`. Gotya. And as for my initial response, you're just making explicit something that was previously implicit. Namely, that `contents` could be `None`.

Gotya!  Lemme look at the test failures then!

Reply:
Yep! Thanks :)